NeoML/include/NeoML/Dnn/DnnSimpleAPI.h

```diff
@@ -0,0 +1,512 @@
+	auto weights = Source( net, "Weightss" );
+
+	auto fc = FullyConnected( 100 );
+
+	x = fc( x ); // 1.
+	y = fc( y ); // 2. Share weights with 1.
+	x = Concat() ( x, y );
+	x = Relu() ( x );
+	x = fc( x );
+	x = Gelu()( x );
+	x = Dropout( 0.5f )( x )
+	x = FullyConnected( 1 )( x );
+	BinaryCrossEntropy( 2.0f )( x, labels, weights );
+
+*/
+
+
+//////////////////////////////////////////////////////////////////////////////////////////
+
+// Create CSourceLayer with name
+NEOML_API CBaseLayer* Source( CDnn& network, const char* name );
+// Create CSinkLayer with name
+NEOML_API CSinkLayer* Sink( const CLayerOutput& layer, const char* name );
+
+NEOML_API CLayerWrapper<CDropoutLayer> Dropout( float dropoutRate,
+	bool isSpatial = false, bool isBatchwise = false );
+
+// Activation functions
+NEOML_API CLayerWrapper<CReLULayer> Relu();
+NEOML_API CLayerWrapper<CGELULayer> Gelu();
+NEOML_API CLayerWrapper<CLinearLayer> Linear( float multiplier, float freeTerm );
+NEOML_API CLayerWrapper<CELULayer> Elu( float alpha = 0.01f );
+NEOML_API CLayerWrapper<CLeakyReLULayer> LeakyRelu( float = 0.01f );
+NEOML_API CLayerWrapper<CAbsLayer> Abs();
+NEOML_API CLayerWrapper<CSigmoidLayer> Sigmoid();
+NEOML_API CLayerWrapper<CTanhLayer> Tanh();
+NEOML_API CLayerWrapper<CHardTanhLayer> HardTanh();
+NEOML_API CLayerWrapper<CHardSigmoidLayer> HardSigmoid( float slope, float bias );
+NEOML_API CLayerWrapper<CPowerLayer> Power( float exponent );
+NEOML_API CLayerWrapper<CHSwishLayer> HSwish();
+
+NEOML_API CLayerWrapper<CFullyConnectedLayer> FullyConnected(
+	int numberOfElements, bool zeroFreeTerm = false );
+
+NEOML_API CLayerWrapper<CBinaryCrossEntropyLossLayer> BinaryCrossEntropyLoss(
+	float positiveWeight = 1.0f, float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CCrossEntropyLossLayer> CrossEntropyLoss(
+	bool isSoftmaxApplied = true, float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CEuclideanLossLayer> EuclideanLoss( float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CHingeLossLayer> HingeLoss( float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CSquaredHingeLossLayer> SquaredHingeLoss(
+	float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CBinaryFocalLossLayer> BinaryFocalLoss( 
+	float focalForce, float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CCenterLossLayer> CenterLoss( int numberOfClasses,
+	float classCentersConvergenceRate, float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CMultiHingeLossLayer> MultiHingeLoss( float lossWeight = 1.0f );
+NEOML_API CLayerWrapper<CMultiSquaredHingeLossLayer> MultiSquaredHingeLoss(
+	float lossWeight = 1.0f );
+
+NEOML_API CLayerWrapper<CPositionalEmbeddingLayer> PositionalEmbedding(
+	CPositionalEmbeddingLayer::TPositionalEmbeddingType type );
+
+NEOML_API CLayerWrapper<CSoftmaxLayer> Softmax(
+	CSoftmaxLayer::TNormalizationArea normalizationArea );
+NEOML_API CLayerWrapper<CAddToObjectLayer> AddToObject();
+
+// Special values for parameters in Reshape
+enum TTransformRule {

```

Comment:
`TTransformRule` isn't used. Looks suspicious...

Reply:
Better write it down in the comment to the `Transform(` function (we have to compile this code with GCC/Clang/VS2015/VS2019, better not to give them cause for warnings)

Reply:
Can you remove this enum plz (mention `-1` and `-2` values in comment)

Reply:
ok